ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -55,7 +55,7 @@ export const EntriesList: React.FC<EntriesListProps> = ({
   const [startTime, setStartTime] = useState(0);
   const [truncatedTimestamp, setTruncatedTimestamp] = useState(0);
 
-  const leftOffBottom = entries.length > 0 ? entries[entries.length - 1].id + 1 : -1;
+  const leftOffBottom = entries.length > 0 ? entries[entries.length - 1].id : "latest";
```

Comment:
I think we need the +1 so we won't have the same entry twice

Reply:
It's now a string. I have moved that to the database server:
 - https://github.com/up9inc/basenine/blob/c741e4aa1cf2a7105d5cc5709d3c1686d66e23db/server/lib/storages/native.go#L1167
 - https://github.com/up9inc/basenine/blob/c741e4aa1cf2a7105d5cc5709d3c1686d66e23db/server/lib/storages/native.go#L396